### PR TITLE
Documentation for processes

### PR DESCRIPTION
**Pull request by antho1404**
https://deploy-preview-68--mesg-docs.netlify.com/guide/orchestrator/

### CODE REVIEW

**Comment by NicolasMahe** [comment]
The list are not aligned correctly in this PR, in the menu but also on any page.

### Before:

Screen Shot 2019-09-19 at 15 38 43

### After:

Screen Shot 2019-09-19 at 15 38 53

**Comment by NicolasMahe** [comment]
@antho1404 check https://github.com/mesg-foundation/docs/pull/72 for the modification did

File: guide/process/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Orchestration system
+
+The orchestration system is an event-driven task-orchestration system that managed the connection between events and tasks of services by following the steps described by processes.
+
+It lets you build and run complex applications by describing their business logic and let the Engine executes them for you.
+
+Processes are an evolution of [Applications](../application/) as they are managed by the Engine and will be distributed on the upcoming decentralized network.
+
+
+__TODO: rename App by Process on the following image:__
```
Comment (antho1404) [inline guide/process/README.md]: TODOs should be removed or commented to not appear in the page

File: guide/process/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Event trigger
+- Task's result trigger
+- Execute a task
+- Condition on the data
+
+Only the first step of a process is and must be an event or a task's result trigger.
+
+### State machine
+
+Processes are [nondeterministic finite](https://en.wikipedia.org/wiki/Nondeterministic_finite_automaton) [state machines](https://en.wikipedia.org/wiki/Finite-state_machine) that perform state transition when matching events or task's results occur but don't support cycle.
```
Comment (antho1404) [inline guide/process/README.md]: Having this in a specific section is a bit weird.

Let's move the content of state machine and activity diagram to the `what is a process`

**Review by NicolasMahe — APPROVED** [review]
Looking good for me!

**Pull request merged**